label_studio/ml_model_providers/models.py

```diff
@@ -27,8 +27,8 @@ class ModelProviderConnection(models.Model):
 
     endpoint = models.CharField(max_length=512, null=True, blank=True, help_text='Azure OpenAI endpoint')
 
-    cached_available_models = models.CharField(
-        max_length=4096, null=True, blank=True, help_text='List of available models from the provider'
+    cached_available_models = models.TextField(
+        null=True, blank=True, help_text='List of available models from the provider'
```

Comment:
@hakan458 could you post the sql that this migration generates using `sqlmigrate`? Not sure if we have precedent for this kind of migration, or how it might affect our deployments. It might actually be safer to create a new TextField for this and deprecate the old field instead of migrating the existing field (though I'm not sure on this point).